README.md

```diff
@@ -54,9 +54,19 @@ RSpec.describe 'Create a work draft', :rack_test do
 
 [Cyperful](https://github.com/stepful/cyperful) is a visual debugger for system tests. To run a system test, prepend `CYPERFUL=1`. For example:
 ```
-CYPERFUL=1 bundle exec rspec spec/system/create_work_draft_spec.rb
+CYPERFUL=1 bin/rspec spec/system/create_work_draft_spec.rb
 ```
 
+### Code Linters
+
+To run all configured linters, run `bin/rake lint`.
+
+To run linters individually, run which ones you need:
+
+* Ruby code: `bin/rubocop`
+* ERB templates: `bin/erb_lint --lint-all --format compact`
+* JavaScript code: `yarn run lint`
+
```

Comment:
Can we also document the format for autofixing (since that is what we will mostly be running)?

Reply:
Done.